fetch.bs

```diff
@@ -4334,7 +4355,8 @@ the response. [[!HTTP-CACHING]]
    <li><p><var>request</var>'s <a for=request>mode</a> is "<code>same-origin</code>",
    "<code>cors</code>", or "<code>no-cors</code>"
 
-   <li><p><var>request</var>'s <a for=request>window</a> is an <a>environment settings object</a>
+   <li><p><var>request</var>'s <a for=request>client</a>'s
```

Comment:
I believe `client` can be null, no? Perhaps there can be a helper algorithm to return request's document or window and performs the appropriate null checks?

Reply:
Yeah... not sure about the helper algorithm as I see people reaching for it a bit too eagerly, but maybe as a followup if we take a look at existing patterns in a bit more detail. Will fix the null dereference, and thanks for catching it.